src/main/java/com/timeaway/repositories/PaidTimeOff.java

```diff
@@ -0,0 +1,51 @@
+package com.timeaway.repositories;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.persistence.*;
+
+import java.sql.Date;
+
+@Entity
+@Table(name = "paid_time_off")
+public class PaidTimeOff {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    private Date startDate;
```

Comment:
I would suggest to use modern java time classes instead, like instant or zonnedtimedate.